internal/driver/fetch_test.go

```diff
@@ -398,27 +398,27 @@ func TestFetchWithBase(t *testing.T) {
 				},
 				{
 					values: []int64{-2700, -608881724},
-					labels: map[string][]string{"pprof::base": {"true"}},
+					labels: map[string][]string{profile.DiffBaseLabelKey: {profile.DiffBaseLabelValue}},
```

Comment:
I probably wouldn't bother using the constants in the tests. It's fine to have some duplication.

Exposing both key and value constants looks noisy. Would it make sense to have function like Sample.DiffBaseSample instead?

Reply:
Done (for both)